README_AWS.md

```diff
@@ -0,0 +1,176 @@
+### Table of Contents ###
+
+- [Setup](#setup)
+  - [VPC](#vpc)
+  - [AWS Environment](#aws-environment)
+  - [IAM Support Role](#iam-support-role)
+- [Available tools](#available-tools)
+  - [1. Egress Verification](#1-egress-verification)
+    - [1.1 Usage](#11-usage)
+      - [1.1.1 CLI Executable](#111-cli-executable)
+      - [1.1.2 AWS Go SDK Implementaiton](#112-aws-go-sdk-implementation)
+    - [1.2 Interpreting Output](#12-interpreting-output)
+    - [1.3 Workflow](#13-workflow)
+  - [2. BYOVPC Configurations Verification](#2-byovpc-configurations-verification)
+
+## Setup ##
+### VPC ###
+- Any VPC for a non-STS CCS cluster can be tested using this tool.
+- You will need to know the following information about the VPC to be verified.
+    - subnet IDs
+    - AWS region
```

Comment:
and credentials?

Reply:
It's the next item in the list, but I switched it to be first to avoid this confusion.